orangecontrib/imageanalytics/image_embedder.py

```diff
@@ -56,11 +56,13 @@ def __init__(self, model, layer, server_url='api.biolab.si:8080'):
         cache_file_path = self._cache_file_blueprint.format(model, layer)
         self._cache_file_path = join(cache_dir(), cache_file_path)
         self._cache_dict = self._init_cache()
-        self._session = cachecontrol.CacheControl(
-            requests.session(),
-            cache=cachecontrol.caches.FileCache(
-                join(cache_dir(), __name__ + ".ImageEmbedder.httpcache"))
-        )
+
+        cached = Memory(
+            cachedir=join(cache_dir(), __name__ + ".ImageEmbedder.httpcache"),
+            compress=1, bytes_limit=100e6).cache
```

Comment:
I think you still have to call `Memory.reduce_size` manually for the bytes_limit to have any effect (at least per joblib's [changelog](https://github.com/joblib/joblib/blob/master/CHANGES.rst).